backend/src/agent/persistence/main.go

```diff
@@ -125,4 +127,5 @@ func init() {
 		"/apis/v1beta1", "The base path for the ML pipeline API server.")
 	flag.StringVar(&namespace, namespaceFlagName, "", "The namespace name used for Kubernetes informers to obtain the listers.")
 	flag.Int64Var(&ttlSecondsAfterWorkflowFinish, ttlSecondsAfterWorkflowFinishFlagName, 604800 /* 7 days */, "The TTL for Argo workflow to persist after workflow finish.")
+	flag.IntVar(&numWorker, numWorkerName, 16, "Number of worker for sync job.")
```

Comment:
Would you still mark it as 2 in this PR? Thanks.

Reply:
kindly ping to mark it as less value while manifest can change it.

Reply:
@rmgogogo You are looking at a historical revision. Current version has updated to 2.